installer/charts/tssc-infrastructure/templates/postgres/pgsql-service.yaml

```
@@ -0,0 +1,127 @@
+        app: {{ $podName }}
+        phase: reference
+    spec:
+      volumes:
+        - name: pgsql-storage
+          persistentVolumeClaim:
+            claimName: {{ $pvcName }}
+      securityContext:
+        runAsNonRoot: true
+        fsGroup: 2000
+      containers:
+      - name: {{ $pgsql.podName }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)